test/SlowTests/Issues/RavenDB_18357.cs

```diff
@@ -15,10 +16,18 @@ public RavenDB_18357(ITestOutputHelper output) : base(output)
     }
 
     [RavenTheory(RavenTestCategory.Querying | RavenTestCategory.Indexes | RavenTestCategory.Corax)]
-    [RavenData(SearchEngineMode = RavenSearchEngineMode.Corax)]
-    public void AutoIndexShouldThrowWhenTryingToIndexComplexObjec(Options options)
+    [RavenExplicitData(SearchEngineMode = RavenSearchEngineMode.Corax)]
```

Comment:
why this needs to be done like that? Can't we just do:

`options.ModifyDatabaseRecord += ...`

or
```
var oldModifyDatabaseRecord = options.ModifyDatabaseRecord;
options.ModifyDatabaseRecord  = doc =>
{
    oldModifyDatabaseRecord(doc);
    doc.Settings[RavenConfiguration.GetKey(x => x.Core.ThrowIfAnyIndexCannotBeOpened)] = "false";
}
```

Reply:
done